src/blenderbim/blenderbim/core/qto.py

```diff
@@ -21,3 +21,22 @@ def calculate_circle_radius(qto, obj=None):
     result = qto.get_radius_of_selected_vertices(obj)
     qto.set_qto_result(result)
     return result
+
+def assign_pset_qto(qto, selected_objects):
+    for object in selected_objects:
+        qto.set_active_object(object)
```

Comment:
Setting active object is slow. I don't think it is necessary to do this. Can you recode it so it doesn't need to do this?

Reply:
Ok, i deleted it. The only side effect is that the pset panel (in Object Property panel) doesn't refresh so it seems that the button doesn't function but it actually works correctly.
Will this problem be resolved refactoring pset module?